website/docs/docs/building-a-dbt-project/building-models/using-custom-schemas.md

```diff
@@ -114,6 +114,8 @@ If you want to use this pattern, you'll need a `generate_schema_name` macro in y
 
 </File>
 
+**Note:** When using this macro, you'll need to set the target name in your job specifically to "prod" if you want custom schemas to be applied.
```

Comment:
I think this is actually dependent on what their target names are. I suspect they have logic in their generate_schema_name macro that says, using the schema if target = `prod`. I think the reminder we actually want would be to say, in dbt Cloud, you need to explicitly declare your target in the job/development configurations.

Reply:
@amychen1776 They copied and pasted the `get_custom_schema.sql` file that we show here. There was no change to the logic in the macro after that.

Reply:
Do you know if they changed the macro above? The `generate_schema_name`?

Reply:
Nope! They literally just copied and pasted it.

Reply:
@amychen1776 This pertains specifically to [An alternative pattern for generating schema names](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-custom-schemas#an-alternative-pattern-for-generating-schema-names). I trip up on this regularly, but when we use the modified one that:
- in `dev` -> puts everything in the `dbt_{username}` schema
- in `prod` -> puts everything in their custom schema (without prefixed `dbt_{username}`)

The only way this works is if you have your prod target called `prod`.

Not sure if there's another workaround for this currently

Reply:
TIL :)